.github/workflows/test-cluster-static-domain-set-up.yml

```diff
@@ -23,8 +23,8 @@ jobs:
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.AWS_STACK_KEY }}
-          aws-secret-access-key: ${{ secrets.AWS_STACK_SECRET }}
+          aws-access-key-id: ${{ secrets.TESTING_ACCOUNT_IAM_KEY }}
+          aws-secret-access-key: ${{ secrets.TESTING_ACCOUNT_IAM_SECRET }}
```

Comment:
Can we keep the same pattern like prod/staging IAM secrets?

AWS_ACCESS_KEY_ID
AWS_SECRET_ACCESS_KEY

AWS_STAGING_ACCESS_KEY_ID
AWS_STAGING_SECRET_ACCESS_KEY

......

Reply:
You mean `AWS_TESTING_ACCESS_KEY_ID`and ` AWS_TESTING_SECRET_ACCESS_KEY`

Reply:
yes